ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/boot-categories-project/provision/cucushift-installer-rehearse-nutanix-ipi-boot-categories-project-provision-chain.yaml

```diff
@@ -9,5 +9,6 @@ chain:
   - chain: ipi-install-nutanix
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-nutanix-vip
```

Comment:
Can we add this step to more install workflow? e.g; disconnected, and proxy ones.

Reply:
Added to disconnected and proxy ones